Feature add group invitations

whassa opened the pull request:
As asked in this feature request: [#37](https://github.com/michael-milette/moodle-enrol_invitation/issues/37)

I've added the functionality to allow users who has the right access to add invitee to groups they are in or that they have control over.

Commit:
Add group invitations

Add install.xml and fix version

michael-milette reviewed (changes requested):
Hi @whassa, 

Thank you for working on this pull request. I think you are missing a change to the install.xml file. Please update your submission, squash your commits, rebase and update your pull request.

Let me know if you have any questions.

Best regards,

Michael Milette

whassa commented:
I have updated the pr to be rebased on the actual master and I've updated the install XML file to also add the groupsid row.
Please let me know if I forgot anything else. 

Thanks!

whassa commented:
Hi @michael-milette, I'm just repinging you to make sure everything is good!

michael-milette commented:
Hi @whassa ,

Thank you for you patience. I am unavailable right now but should be able to look at this again in a few weeks.

Best regards,

Michael
